crates/red_knot_python_semantic/src/semantic_index.rs

```diff
@@ -33,7 +32,7 @@ mod use_def;
 
 pub(crate) use self::use_def::{
     BindingWithConstraints, BindingWithConstraintsIterator, DeclarationWithConstraint,
-    DeclarationsIterator,
+    DeclarationsIterator, UseDefMap,
```

Comment:
Where do we end up needing to use this from outside the semantic index in this PR?